docs/news/news-v5.0.x.rst

```diff
@@ -155,13 +160,14 @@ Changes since rc4:
   - Open MPI now uses readthedocs.io for all documentation.
   - Converted man pages to markdown. Thanks to Fangcong Yin for their contribution
     to this effort.
-  - Various ``README.md`` fixes - thanks to: Yixin Zhang, Samuel Cho,
-    Robert Langfield, Alex Ross, Sophia Fang, mitchelltopaloglu, Evstrife,
-    and Hao Tong for their contributions.
+  - Various ``README.md`` and ``HACKING.md`` fixes - thanks to: Yixin Zhang, Samuel Cho,
+    Robert Langfield, Alex Ross, Sophia Fang, mitchelltopaloglu, Evstrife, Hao Tong
+    and Lachlan Bell for their contributions.
   - Various CUDA documentation fixes. Thanks to Simon Byrne for finding
     and fixing these typos.
 - Build updates and fixes:
```

Comment:
Missing a blank line before this one, too.

Reply:
Ugh - forgot about that syntax. thanks.